docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,100 @@
+JDBC Poller Input Operator
+=============
+
+## Operator Objective
+This operator scans jdbc table in parallel fashion. Based on requrested partition count, number of non polling static partitions are created to scan table portitions in parallel. Additional single partition is poller partition which scans for new data incrementally. Assumption is that there is an ordered column using which range queries can be formed. Only newly added data will be fetched by the polling. Also out of order insertion/deletion won't be supported.
+
+JDBC Poller is **idempotent**, **fault-tolerant** and **statically partitionable**.
+
+## Operator Usecase
+1. Scan huge database tables to either copy to other DB or process it using **Apache Apex**
```

Comment:
Define DB before using it, or use ```database``` instead.
Please also give an example for the use case.